dicp/dicp/vendor/TopsGraph/conversion.py

```diff
@@ -210,14 +210,10 @@ def HardswishBackward(self, *args, **kwargs):
     def Clone(self, *args, **kwargs):
         return self.get_proxy(tops_op.Clone, args, kwargs)
 
-    @register_conversion(aten.copy.default)
+    @register_conversion([aten.copy.default, aten.copy_.default])
```

Comment:
add comment that warns for situation besides inplace copyting the input args.

Reply:
done.